doc/examples/affine_registration_3d.py

```diff
@@ -0,0 +1,236 @@
+"""
+==========================================
+Affine Registration in 3D
+==========================================
+This example explains how to compute an affine transformation to register two 3D
```

Comment:
Hi @omarocegueda in many places in this example you are violating the line size limit for PEP8. It's at 79 characters. The 80th character should change line.